utils/backport.sh

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env bash
+set -x
+set -e
+
+commit_to_backport=$1
+
+git fetch upstream
+
+
+function backport() {
+    branch=${1}
+    echo "Backporting ${commit_to_backport} to ${branch}.."
+    git checkout ${branch}
+    git merge upstream/${branch}
```

Comment:
in my version, I delete the branch, then check it out
to make sure I don't have merge issue when someone pull commits out, or that my local branch isn't messed up.
```suggestion
    git branch -D ${branch} || echo "no branch"
    git checkout upstream/${branch} --track
```